lib/vehicle.rb

```diff
@@ -5,14 +5,24 @@ class Vehicle
               :year,
               :make,
               :model,
-              :engine
-
+              :engine,
+              :registration_date,
+              :plate_type
+  attr_accessor :vin,
+                :year,
+                :make,
+                :model,
+                :engine,
+                :registration_date,
+                :plate_type
```

Comment:
Same comment as before -- selectively set attributes as accessors. I believe the only attributes you need to be able to update outside of this class are `registration_Date` and `plate_type` so those should be the only accessors, and the rest could be readers.

Reply:
Yes I should have cleaned this up. Sorry!